### DISCUSSION TITLE
how to connect to table with custom name, but known schema

### DISCUSSION THREAD

**Discussion by g992** [prisma/prisma#20860] [Q&A]
### Question

I describe model in schema.prisma, e.g.:
```
model Test {
  id        Int       @id @default(autoincrement())
  foo       String    @db.VarChar(255)
  bar       String    @db.VarChar(255)
}
```
and i want to create tables with different names e.g.: 'testTable1', testTable2' with schema Test, and use it by `prisma['testTable1'].create(...)`

In mongose it looks like 
```
const connection = connection.model('testTable1', testSchema)
connection.create(...)
```

### Information about Prisma Schema, Client Queries and Environment (optional)

- OS: node18-alpine docker image 
- Database: PostgreSQL
- Node.js version: 18.14.0

- Prisma version 5.2.0

**Comment by nurul3101** [comment]
Hello @g992 👋 

Do you mean you want to create tables dynamically based on a certain model?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
